shared/actions/chat/index.js

```diff
@@ -955,6 +983,7 @@ function* chatSaga(): SagaGenerator<any, any> {
   yield Saga.safeTakeEvery('chat:updateBadging', _updateBadging)
   yield Saga.safeTakeEvery('chat:updateInboxComplete', _ensureValidSelectedChat, false, false)
   yield Saga.safeTakeEvery('chat:updateMetadata', _updateMetadata)
+  yield throttle(5000, 'chat:updateTyping', _updateTyping)
```

Comment:
i think we should be careful about this throttle. first this is 5000 ms and not 500 (as described in the pr). second we likely don't want to throttle it if the value is '' (we want to clear immediately). i think the delay should be larger, likely 2-3 seconds. Also this'll likely throttle across mulitple conversations (maybe thats impossible practically but i think throttle isn't going to introspect the parameters).
it might make more sense to throttle the dispatch action instead of the saga here. lets discuss